external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceProvider.scala

```diff
@@ -457,10 +457,14 @@ private[kafka010] class KafkaSourceProvider extends DataSourceRegister
 
 private[kafka010] object KafkaSourceProvider extends Logging {
   private val STRATEGY_OPTION_KEYS = Set("subscribe", "subscribepattern", "assign")
+  private val ASSIGN = "assign"
+  private val SUBSCRIBEPATTERN = "subscribepattern"
+  private val SUBSCRIBE = "subscribe"
```

Comment:
If we have all individual values, we can change `STRATEGY_OPTION_KEYS` together, can't we?
```
- private val STRATEGY_OPTION_KEYS = Set("subscribe", "subscribepattern", "assign")
+ private val STRATEGY_OPTION_KEYS = Set(SUBSCRIBE, SUBSCRIBEPATTERN, ASSIGN)
```

Reply:
It looks better.